test/unit/definitions/divorce/CaseEventToFields.test.js

```diff
@@ -1,36 +1,108 @@
+    .concat(load('definitions/divorce/json/CaseEvent/CaseEvent-general-order-nonprod.json'))
+    .concat(load('definitions/divorce/json/CaseEvent/CaseEvent-pet-amend-nonprod.json'))
+    .concat(load('definitions/divorce/json/CaseEvent/CaseEvent-nonprod.json'));
+
+  return [...caseEvent, ...definitions];
+}
+
+function mergeCaseEventJsonProdFiles() {
+  const definitions = []
+    .concat(load('definitions/divorce/json/CaseEvent/CaseEvent-prod.json'));
+
+  return [...caseEvent, ...definitions];
+}
+
+function mergeCaseFieldJsonNonProdFiles() {
+  const definitions = []
+    .concat(load('definitions/divorce/json/CaseField/CaseField-deemed-and-dispensed-nonprod.json'))
+    .concat(load('definitions/divorce/json/CaseField/CaseField-general-order-nonprod.json'))
+    .concat(load('definitions/divorce/json/CaseField/CaseField-pcq-nonprod.json'));
+
+  return [...caseField, ...definitions];
+}
+
+function mergeCaseFieldJsonProdFiles() {
+  const definitions = []
+    .concat(load('definitions/divorce/json/CaseField/CaseField-prod.json'));
+
+  return [...caseField, ...definitions];
+}
+
+function mergeCaseEventToFieldsJsonNonProdFiles() {
+  const definitions = []
+    .concat(load('definitions/divorce/json/CaseEventToFields/CaseEventToFields-deemed-and-dispensed-nonprod.json'))
+    .concat(load('definitions/divorce/json/CaseEventToFields/CaseEventToFields-general-order-nonprod.json'))
+    .concat(load('definitions/divorce/json/CaseEventToFields/CaseEventToFields-nonprod.json'));
+
+  return [...caseEventToFields, ...definitions];
+}
+
+function mergeCaseEventToFieldsJsonProdFiles() {
+  const definitions = []
+    .concat(load('definitions/divorce/json/CaseEventToFields/CaseEventToFields-prod.json'));
+
+  return [...caseEventToFields, ...definitions];
+}
+
+function assertHasOnlyValidEventIds(caseEventToFieldsFile, caseEventFile) {
+  const errors = [];
+  caseEventToFieldsFile.forEach(caseEventToFieldsEntry => {
+    try {
+      expect(find(caseEventFile, ['ID', caseEventToFieldsEntry.CaseEventID])).to.be.an('object');
+    } catch (error) {
+      errors.push(`Event ID ${caseEventToFieldsEntry.CaseEventID} is not valid`);
     }
   });
-  it('should contain valid field IDs', () => {
-    const errors = [];
-    caseEventToFields.forEach(caseEventToFieldsEntry => {
-      try {
-        expect(find(caseField, ['ID', caseEventToFieldsEntry.CaseFieldID])).to.be.an('object');
-      } catch (error) {
-        errors.push(`Field ID ${caseEventToFieldsEntry.CaseFieldID} is not valid`);
-      }
-    });
-    if (errors.length) {
-      assert.fail(`Found invalid field IDs - ${errors}`);
+  if (errors.length) {
+    assert.fail(`Found invalid case IDs - ${errors}`);
+  }
+}
+
+function assertHasOnlyValidFieldIds(caseEventToFieldsFile, caseFieldFile) {
+  const errors = [];
+  caseEventToFieldsFile.forEach(caseEventToFieldsEntry => {
+    try {
+      expect(find(caseFieldFile, ['ID', caseEventToFieldsEntry.CaseFieldID])).to.be.an('object');
```

Comment:
object, not string?

Reply:
Hmm, I just copied from the other file didn't pay attention but I just tried with 'string' now and it fails